src/main/resources/anniv5Resources/localization/eng/anomalypack/Cardstrings.json

```diff
@@ -0,0 +1,59 @@
+{
+  "${ModID}:Wonder": {
+    "NAME": "Wonder",
+    "DESCRIPTION": "Discard your hand. NL Draw a card and set your Energy to its cost.",
+    "UPGRADE_DESCRIPTION": "Discard your hand. Draw a card and set your E to its cost. NL Retain."
+  },
+  "${ModID}:FreeSpace": {
+    "NAME": "Free Space",
+    "DESCRIPTION": "When you draw, discard, exhaust or play this, draw a card. NL Ethereal.",
+    "UPGRADE_DESCRIPTION": "When you draw, discard, exhaust or play this, draw a card."
+  },
+  "${ModID}:MidnightOil": {
+    "NAME": "Midnight Oil",
+    "DESCRIPTION": "Exhaust a random card. Gain [E]. While this is Exhausted, you gain 1 [E] each turn.",
+    "UPGRADE_DESCRIPTION": "Exhaust a card. Gain [E]. While this is Exhausted, you gain 1 [E] each turn."
```

Comment:
No need to specify 1
Does [E] show up correctly if you have a period right after it?